scss/bootstrap/overrides/style/_thumbnail.scss

```diff
@@ -0,0 +1,75 @@
+.crm-mosaico-template-item {
+  background: $crm-background-light;
+  border: 2px solid $crm-white !important;
+  @include prefix(border-radius, 4px, moz webkit ms);
+  margin: auto;
+  opacity: 1.0;
+  width: 165px;
+
+  .crm-mosaico-template-image-wrapper {
+    background: $crm-dark-gray;
+    height: 200px;
+    overflow: hidden;
+    position: relative;
+
+    img {
+      height: auto !important;
+    }
+
+    .crm-mosaico-template-actions-wrapper {
+      bottom: 0;
+      display: table;
+      height: 100%;
+      left: 0;
+      position: absolute;
+      right: 0;
+      text-align: center;
+      top: 0;
+      width: 100%;
+    }
+
+    .crm-mosaico-template-actions {
```

Comment:
Its already sorted. Current plugins sort @include based on the attribute inside it. Is it incorrect?